### PR TITLE
[IMP] Website: GTM integration

### PR DESCRIPTION
**Pull request by masp-odoo**
taskID-4006105

### CODE REVIEW

**Comment by masp-odoo** [comment]
Hey @auva-odoo, here's an exponation from APL if it can help:
_"Odoo integrates well with GTM, you can use GTM and manage all the tags from GTM. What we're talking about here is adding code to odoo (in the form of a datalayer) to make the information available to GTM so that it can be sent to analytics. We don't do that. But we do allow GTM to be integrated in full standard via code injection. Here, when we talk about datalayers, we're really talking about adding, for example, the name, price, taxes, etc. for each product in a specific datalayer code so that GTM can send the information back to analytics. We don't do that at the moment.
Some GTM tags use a data layer to retrieve variables and send them to Google Analytics. This information is not currently managed in Odoo. Which in itself is quite logical. This could be done (check with the POs for this), but to use them, the customer would have to put GTM on their site and then configure their tag correctly. Now, if users want to track advanced ecommerce in Google Analytics, that's not possible at the moment. This represents a small proportion of our users.

With GTM, we can already do a lot of things outside of this tag, so the standard integration works."_

File: content/applications/websites/website/reporting/analytics.rst [hunk 1]
```diff
@@ -137,7 +137,9 @@ measurement codes and related code fragments, collectively known as tags on your
 app, directly through the code injector.
 
 .. warning::
-   Google Tag Manager may not be compliant with local data protection regulations.
+   - GTM integration is possible in Odoo standard, although some data layers might not be installed
+     (e.g. advanced eCommerce tracking data layers).
```
Comment (auva-odoo) [inline content/applications/websites/website/reporting/analytics.rst:141]: What if we said this instead (copied from your explanation); I think by providing a little bit more context and explanations, it makes it easier for people to understand?

```suggestion
   - Some GTM tags use data layers (e.g., advanced eCommerce tracking data layers) to retrieve variables and send them to Google Analytics. Data layers are currently not managed in Odoo. 
```
(I changed it a little bit, just trying to make it as complete as possible while staying as concise as possible; is it still accurate?)

**Review by auva-odoo — CHANGES_REQUESTED** [review]
@masp-odoo Little suggestion for you, what do you think?

File: content/applications/websites/website/reporting/analytics.rst (hunk 1 above)
Comment (masp-odoo) [reply content/applications/websites/website/reporting/analytics.rst:141]: Hi, it seems accurate to me :) !

**Review by auva-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**